app.js

```diff
@@ -1,10 +1,18 @@
 const express = require("express");
 const app = express();
-const { getCategories } = require("./controllers/controllers.js");
+const { getCategories, getReviews } = require("./controllers/controllers.js");
```

Comment:
Tip: as you complete more endpoints you may wish to split your controllers into catergoriesControllers.js / reviewsControllers.js
There is nothing wrong with this filename for now, but its useful to think about how we might name files even more descriptively.